test/runner/lib/core_ci.py

```diff
@@ -106,6 +106,7 @@ def __init__(self, args, platform, version, stage='prod', persist=True, name=Non
 
                 display.info('Loaded existing %s/%s instance %s.' % (self.platform, self.version, self.instance_id),
                              verbosity=1)
+                display.info('Instance %s/%s on %s' % (self.platform, self.version, self._uri), verbosity=1)
```

Comment:
Ok, I did not want to make the line even longer (risking it to be out of sight).
Fixed.